data/data/agent/files/usr/local/bin/start-agent.sh.template

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Read the image from the  environment file since it was not set up when service was started
+export PULL_SECRET_TOKEN={{.PullSecretToken}}
```

Comment:
Is this required by `agent`? If so, couldn't be passed in other mode? In this way looks like a potential way for leaking it (ie logs captured via journalctl)

Reply:
> Is this required by `agent`? If so, couldn't be passed in other mode? In this way looks like a potential way for leaking it (ie logs captured via journalctl)

Let me test without it, I don't think this is needed, I had copied it from the service but the prior start-agent.sh didn't need it

Reply:
Tested and removed the PULL_SECRET_TOKEN setting.